repositories.yaml

```diff
@@ -2068,3 +2068,23 @@ applications:
     channels:
       - v1_name: subscription-platform-backend
         app_id: subscription.platform.backend
+
+  - app_name: experimenter_cirrus
+    canonical_app_name: "Experimenter"
+    app_description: >
+      Experimenter's instances of Cirrus
```

Comment:
Could you go a little into what "Experimenter" and "Cirrus" mean here? This description will be public in e.g. Glean Dictionary, so it might pay to have this require less insider information.

Reply:
Yes, can do!